### PR TITLE
closes #102 - tos page

### LINKED ISSUE #102: terms of services page
### Problem

We have a link to a TOS but no page

### Solution

Create the page

### PR DESCRIPTION
**Pull request by taoeffect**

### CODE REVIEW

File: src/pages/terms-and-conditions.astro [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    font-weight: 600;
+    margin: 3rem auto 2rem auto;
+  }
+
+  p {
+    margin-bottom: 1.5rem;
+  }
+}
+
+.c-container-center {
```
Comment (SebinSong) [inline src/pages/terms-and-conditions.astro]: This local class doesn't seem to be in use and can be removed.

File: src/pages/terms-and-conditions.astro [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+.c-main-title {
+  max-width: 47rem;
+  margin: 1.5rem auto;
+
+  @include phone {
+    max-width: 30rem;
+  }
+}
+
+.c-line {
```
Comment (SebinSong) [inline src/pages/terms-and-conditions.astro]: same for this one.

**Review by SebinSong — APPROVED** [review]
@taoeffect Looks good to me, other than two comments I left.

**Comment by dotmacro** [comment]
My review submitted via PR #109

**Pull request merged**